### PR TITLE
wishlist-feature-18

### PR DESCRIPTION
**Pull request by vanessa-abewe**
What does this PR do?

- a post request to add an item to the wishlist

Description of Task to be completed?

- The item should be added to the buyer's wishlist

- The product details (name, price, image) should be retrieved from the database

- The updated wishlist should be stored in the buyer's account

- A confirmation message should be sent to the frontend to be displayed to the buyer.

Trello Card : https://trello.com/c/7b3NFqBd

### CODE REVIEW

**Review by Muhet — APPROVED** [review]
nice work

**Review by bryan936M — APPROVED** [review]
Fantastic work

**Review by Muhet — APPROVED** [review]
nice work abewe

**Review by NTElissa — APPROVED** [review]
good progress

File: src/services/user.services.js [hunk 1]
```diff
@@ -11,11 +11,7 @@ export const registerGoogle = async (data) => {
   }
 };
 //getUserByEmail
-<<<<<<< HEAD
 export const getUserByEmail = async (email) => {
-=======
-const getUserByEmail = async (email) => {
->>>>>>> dff5979 (ft(update-product-from-collection-33): Added new function.)
```
Comment (nili-alu) [inline src/services/user.services.js]: can you remove line 13, 16, 18

**Review by nili-alu — APPROVED** [review]

**Review by ALVINdimpos — APPROVED** [review]

**Pull request merged**